Assets/MixedRealityToolkit/Interfaces/InputSystem/Handlers/IMixedRealityPointerUpdatedHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine.EventSystems;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Interface to implement to react to simple pointer input and pointer updates.
+    /// </summary>
+    public interface IMixedRealityPointerUpdatedHandler : IMixedRealityPointerHandler
```

Comment:
I like IMixedRealityPointerDragHandler. What are your thoughts on just adding OnPointerUpdated to the IMixedRealityPointerHandler interface? I can see a downside of this being a breaking change. The upside is it does not add another interface to MRTK.

Reply:
I would like to add it to IMixedRealityPointerHandler (OnPointerDragged?) but it is a breaking change and I don't know how much wiggle room we have there. It is the best type of breaking change though, in the sense that in produces a build time error (missing implementation) that is very straightforward to fix. @wiwei , any thoughts on that?

Reply:
So we're in an odd state. Because we haven't clearly defined the delineation between what is internal vs what is external, technically speaking every single change that we make is technically a breaking change, because someone could have taken a dependency on a utility class that is nested deep within our code, but is still technically public because we distribute as scripts.

This is a problem that we need to solve, but I figured it was interesting to write here because it highlights that there won't be a clear easy answer due to that.

That being said, because of the timing of where we are right now, it feels we might still have some time to do the right long term thing (i.e. IMixedRealityPointerHandler), assuming that we make it very explicit (i.e. we write a doc that highlights a limited set of changes between RC1->) maybe this is okay. We can followup with product on this to see what the agreement is.

Reply:
Okay circling back here, it sounds like we're comfortable with making this one change given the tradeoffs (i.e. no incurring immediate interface technical debt going forward), and we will author a changes doc (in release notes) that describes this change, how it will surface as an error, and provide clear guidance on how to fix it.